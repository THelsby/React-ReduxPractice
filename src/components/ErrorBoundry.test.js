import { shallow } from "enzyme";
import React from "react";
import ErrorBoundry from "./ErrorBoundry";

it("Expect to render ErrorBoundry component", () => {
  expect(shallow(<ErrorBoundry />)).toMatchSnapshot();
});
