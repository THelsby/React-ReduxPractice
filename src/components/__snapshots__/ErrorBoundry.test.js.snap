// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render ErrorBoundry component 1`] = `ShallowWrapper {}`;
